IaC/pxb.cd/init.groovy.d/cloud.groovy

```diff
@@ -215,7 +213,7 @@ initMap['min-buster-x64'] = '''
 initMap['min-bionic-x64'] = '''
     set -o xtrace
     if ! mountpoint -q /mnt; then
-        DEVICE=$(ls /dev/xvdd /dev/xvdh /dev/nvme0n1 | head -1)
+        DEVICE=$(ls /dev/xvdd /dev/xvdh /dev/nvme1n1 | head -1)
```

Comment:
Why this change appeared?

Reply:
Because I've changed AMI ID to newer one and I got an error that nvme0n1 being used

Reply:
can we make it like this 
```
DEVICE=$(ls /dev/xvdd /dev/xvdh /dev/nvme0n1 /dev/nvme1n1 | head -1)
```

Reply:
yeah the suggested approach should be better as we always face this issue and change device name back and forth

Reply:
It won't work, as it'll pick the first available device, then will try to create an ext2 FS
and will stuck on creation